packages/relay/tests/lib/mirrorNodeClient.spec.ts

```diff
@@ -60,7 +60,7 @@ describe('MirrorNodeClient', async function () {
   describe('handleError', async() => {
 
     const CONTRACT_CALL_ENDPOINT = 'contracts/call';
-    const nullResponseCodes = [400,404,415,429,500];
+    const nullResponseCodes = [404,415,500];
```

Comment:
Q: should you be adding 400 and 429 to `errorRepsonseCodes`?

Reply:
✅  Done. Added them.